saleor/core/tasks.py

```diff
@@ -48,3 +55,21 @@ def delete_event_payloads_task(expiration_date=None):
 def delete_files_from_storage_task(paths):
     for path in paths:
         default_storage.delete(path)
+
+
+@app.task
```

Comment:
See my comment above. If we want to delete files in celery task, this task should have retries (in case of an issue with connection with storage) and should take into account that the provided path could not exist (assuming a case, where we have 10 paths, we deleted 2 of them, then received an error. Celery task should retry and logic should be able to handle a case when path doesn't exist) (not sure if .delete will raise exception in that case or handle it silently).
Also IMO, in case all retires tries were not able to delete the files we should raise an exception with a proper message that would allow us to find these files manually.

Reply:
@korycins I've checked Django storages - they ignore non-existing files quietly so there's no need to implement any exection handling here